src/main/java/hunternif/mc/atlas/ClientProxy.java

```diff
@@ -1,7 +1,43 @@
 package hunternif.mc.atlas;
 
-import static hunternif.mc.atlas.client.StandardTextureSet.*;
-import static net.minecraft.world.biome.BiomeGenBase.*;

```

Comment:
Shit, I didn't notice the giant list show up when I auto-imported what I needed...